packages/core/content-releases/admin/src/services/utils.ts

```diff
@@ -2,6 +2,9 @@ import { getFetchClient } from '@strapi/helper-plugin';
 
 import type { AxiosError, AxiosRequestConfig, AxiosResponse } from 'axios';
 
+/* -------------------------------------------------------------------------------------------------
+ * Axios data
+ * -----------------------------------------------------------------------------------------------*/
 export interface QueryArguments<TSend> {
   url: string;
   method: 'PUT' | 'GET' | 'POST' | 'DELETE';
```

Comment:
GitHub won't let me comment on the right line, but I think in the axiosBaseQuery function you can change the if statements to a switch